tensorflow/lite/java/ovic/src/test/java/org/tensorflow/ovic/OvicClassifierTest.java

```diff
@@ -123,7 +123,8 @@ public void ovicClassifier_latencyNotNull() throws Exception {
   public void ovicClassifier_mismatchedInputResolutionFails() throws Exception {
     classifier = new OvicClassifier(labelsInputStream, lowResModel);
     int[] inputDims = classifier.getInputDims();
-    assertThat((inputDims[1] == 128) && (inputDims[2] == 128)).isTrue();
+    assertThat(inputDims[1] == 128).isTrue();
```

Comment:
this can be `assertThat(inputDims[1]).isEqualTo(128)`, which will improve the error message

Reply:
Hi, again, James.

I am sorry that I did not earlier realize that you were using a github feature that allowed your proposed code change to be merged.  I am going to attempt to accept your proposed piggyback patch by clicking "Resolve conversation."  In case that does not work, please note that it was my intent to accept your proposed patch, and please feel free to take whatever action you think is best to move that process forward.

Reply:
You will need to resolve the conflicts that I've created, see https://help.github.com/en/articles/resolving-a-merge-conflict-on-github for help. If that doesn't work, there is a link to "Resolving a merge conflict using the command line" in that page.

This is just a straightforward git merge conflict, it should be easy to resolve.

Reply:
I have gone through that process and, toward the bottom of the web page on which I am writing this comment, it now says "Merging can be performed automatically with 1 approving review."  So, I am assuming I should not take further action at this time, but please let me know if I should do something.  Thanks again for your help, James.   --Adam